t/09-job_clone.t

```diff
@@ -93,7 +94,21 @@ subtest 'get job' => sub {
     qr/failed to get job '$job_id'/, 'invalid job id results in error';
 
     $job_id = 99937;
-    lives_ok { clone_job_get_job($job_id, $remote, $remote_url, \%options) } 'got job';
+    combined_like { clone_job_get_job($job_id, $remote, $remote_url, \%options) } qr/^$/, 'got job';
+};
+
+subtest 'get job with verbose output' => sub {
+    my $config = tempdir;
+    $config->child("client.conf")->touch;
+    $ENV{OPENQA_CONFIG} = $config;
+    my $temp_assetdir = tempdir;
+    my %options       = (dir => $temp_assetdir, host => $host, from => $host, verbose => 1);
```

Comment:
why the `$ENV{OPENQA_CONFIG}` and `$temp_assetdir` are set here?  If I understand correctly, the functions `create_url_handler` and `clone_job_get_job` don't need them.

Reply:
I added those because otherwise these code paths are skipped. The regression with logging in `create_url_handler` could only reproduced with a `.conf` file in place. So this is not needed to pass the test, but it's necessary to have better coverage. Otherwise a regression would not be caught by this test.

Reply:
I suppose I'll add a comment to point this out. Thank you for asking.